### PR TITLE
Revert changes to fix build issues when using the plugin

### PR DESCRIPTION
**Pull request by J-Lou**
Fixing this error message when building on gateway-developer-example.

Failed to apply plugin [id 'com.ca.apim.gateway.gateway-export-plugin']
Could not create an instance of type com.ca.apim.gateway.cagatewayexport.tasks.export.GatewayConnectionProperties_Decorated.
Could not find any public constructor for class com.ca.apim.gateway.cagatewayexport.tasks.export.GatewayConnectionProperties_Decorated which accepts parameters [org.gradle.api.internal.project.DefaultProject_Decorated].

### CODE REVIEW

File: gateway-export-plugin/src/main/java/com/ca/apim/gateway/cagatewayexport/tasks/export/GatewayConnectionProperties.java [hunk 1]
```diff
@@ -14,11 +14,13 @@
...
     private Property<String> userName;
     private Property<String> userPass;
+    private Property<String> folderPath;
 
-    GatewayConnectionProperties(Project project) {
+    public GatewayConnectionProperties(Project project) {
         url = project.getObjects().property(String.class);
         userName = project.getObjects().property(String.class);
         userPass = project.getObjects().property(String.class);
+        folderPath = project.getObjects().property(String.class);
```
Comment (amoghsa) [inline gateway-export-plugin/src/main/java/com/ca/apim/gateway/cagatewayexport/tasks/export/GatewayConnectionProperties.java]: Don't revert whole thing.. Story was about moving folderPath to ExportPluginConfig.. You just need public constructor..

Comment (J-Lou) [reply gateway-export-plugin/src/main/java/com/ca/apim/gateway/cagatewayexport/tasks/export/GatewayConnectionProperties.java]: From what I've initially seen, removing the folderpath from GatewayConnection causes the following issue.

`* What went wrong:
A problem occurred evaluating project ':business-integrations'.
> Could not set unknown property 'folderPath' for object of type com.ca.apim.gateway.cagatewayexport.tasks.export.GatewayConnectionProperties.`

Comment (yves-chan) [reply gateway-export-plugin/src/main/java/com/ca/apim/gateway/cagatewayexport/tasks/export/GatewayConnectionProperties.java]: @J-Lou There was PR #139 which moved the folderPath from GatewayConnectionProperties to GatewayExportConfig, so defining folderPath there should fix your problem. @amoghsa can you update the wiki for the change? Also there was a soap service change too, can you also update the wiki on that section?

Comment (amoghsa) [reply gateway-export-plugin/src/main/java/com/ca/apim/gateway/cagatewayexport/tasks/export/GatewayConnectionProperties.java]: Darn, I did know there were docs as well.. I will get to it..

Comment (amoghsa) [reply gateway-export-plugin/src/main/java/com/ca/apim/gateway/cagatewayexport/tasks/export/GatewayConnectionProperties.java]: Soap service updates have been added..

**Pull request merged**